File: python/prograimg_python_basic/Exercises/conditional_statements/conditional_statements_advanced_-_exercise/hotel_room.py
months = str(input())
days = int(input())

studio = 0
apartment = 0
discount = 0
end_sum = 0
discount_studio = 0

if months == "May" or months == "October":
    studio = 50
    apartment = 65

    if 7 < days < 14:
        discount_studio = (studio * days) * 0.05
    elif days > 14:
        discount_studio = (studio * days) * 0.30

elif months == "June" or months == "September":
    studio = 75.20
    apartment = 68.70

    if days > 14:
        discount_studio = (studio * days) * 0.20

elif months == "July" or months == "August":
    studio = 76
    apartment = 77


if days > 14:
    discount = (apartment * days) * 0.10
    apartment = (apartment * days) - discount

else:
    apartment = apartment * days

end_sum = (studio * days) - discount_studio

print(f"Apartment: {apartment:.2f} lv.")
print(f"Studio: {end_sum:.2f} lv.")

File: JS/JavaScript/JS-Basics/ConditionalStatementsAdvancedExercise/HotelRoom.js
function hotel(inputs) {

    let month = String(inputs[0]);
    let days = Number(inputs[1]);

    let amountForStudio;
    let amountForApartment;

    if (month == "May" || month == "October") {
        amountForStudio = days * 50;
        amountForApartment = days * 65;

        if (days > 7 && days <= 14) {
            amountForStudio -= amountForStudio * 0.05;
        }
        else if (days > 14){
            amountForStudio -= amountForStudio * 0.30;
        };

        if (days > 14) {
            amountForApartment -= amountForApartment * 0.10;
        };
    }
    else if (month == "June" || month == "September") {
        amountForStudio = days * 75.20;
        amountForApartment = days * 68.70;

        if (days > 14) {
            amountForStudio -= amountForStudio * 0.20;
        };

        if (days > 14) {
            amountForApartment -= amountForApartment * 0.10;
        };
    }
    else if (month == "August" || month == "July") {
        amountForStudio = days * 76 ;
        amountForApartment = days * 77;

        if (days > 14) {
            amountForApartment -= amountForApartment * 0.10;
        };

    }

    console.log(`Apartment: ${amountForApartment.toFixed(2)} lv.`)
    console.log(`Studio: ${amountForStudio.toFixed(2)} lv.`)

}


hotel(["May",
"15"])
hotel(["June",
"14"])
hotel(["August",
"20"])
